README.md

```diff
@@ -1,43 +1,160 @@
----
-tags:
-level:
-languages:
-resources:
----
+# Ruby Object Remembrance 
 
-# Ruby Remembering Objects
+## Objectives
 
-## Outline
+1. Understand the concept of remembrance in object-oriented programming. 
+2. Learn how to use class variables to, remember, or store, instances of a class that are produced. 
 
-Remembering objects
+## Introduction
 
-We know a lot about OO
-1. Classes
-2. Instances
-3. Initialize
-4. Self / class vs instance scope
-5. Class variables
-6. Class methods
+Let's say we're building a command line game in which players play various rounds until a final tally determines the winner. Or creating an app in which we want to store a list of all of the users who sign up. Or building a program that helps users track and store the passwords for their various accounts. 
 
-With that we can do something really cool.
+In all of these situations, and many more we can imagine, our application needs a way to store or remember a collection of class instances. Whether they are instances of a `Game`, `User` or `Password` class, all of these examples would require our program to keep track of instances that are created. 
 
-Imagine wanting to build a music manager that kept track of all the songs you entered and let you browse them. Later we'll add more functionality to the application, but for now we'll keep it simple.
+Luckily for us, Ruby allows us to do so by uses class variables to store new instances as soon as they are created. Let's take a look together. 
 
-We've provided the base functionality of entering a song and creating instances, but what's missing is the core functionality of then being able to list out songs.
+## Using class variables to store class instances
```

Comment:
No need to call them "class instances" they are generally just called instances or "instances of a class"